### PR TITLE
dedup v2 playbook

### PR DESCRIPTION
**Pull request by mayagoldb**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/20747

## Description
Dedup v2 playbook. Changes include:
- Task inputs are based on playbook inputs and not hard-coded
- Option to link instead of close the duplicate incidents (also input based).
- Descriptions, etc.

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by mayagoldb** [comment]
@yaron-libman - This is a v2 playbook for dedup, the main changes are in the flow and the playbook's description.

File: Playbooks/playbook-Dedup_-_Generic_v2.yml [hunk 1]
```diff
@@ -0,0 +1,584 @@
...
+  value: {}
+  required: false
+  description: 'A comma-separated list of similar label keys. Comma separated value.
+    Also supports allowing X different words between labels, within the following
+    way: label_name:X, where X is the number of words. X can also be ''*'' for contains.
+    For example: the value "Email/subject:*" will consider email subject similar,
+    if one is substring of the other. Relevant for ''Rules'' method.'
+- key: 'similarIncidentFields '
+  value:
+    simple: name,type
```
Comment (erezh31) [inline Playbooks/playbook-Dedup_-_Generic_v2.yml]: Maybe we should consider add "details" too? @michalgold what do you think?

File: Playbooks/playbook-Dedup_-_Generic_v2.yml [hunk 2]
```diff
@@ -0,0 +1,584 @@
...
+        "x": 10,
+        "y": -40
+      }
+    }
+  }
+inputs:
+- key: DuplicateMethod
+  value: {}
+  required: true
+  description: Select a method for identifying duplicate incidents. Can be "ml", "rules",
```
Comment (erezh31) [inline Playbooks/playbook-Dedup_-_Generic_v2.yml]: I think It's better to specify a little bit details here:
rules - define specific rules, like similar incident fields & labels. It's the best method if you know the exact logic to find similar incidents.
text - text similarity, based on TF-IDF - unique word frequency in the incidents (based on similar incident fields)
ml - machine learning model, was trained on similar phishing incidents. consider similar labels, incident fields and indicators.

We need some doc help here

Comment (michalgold) [reply Playbooks/playbook-Dedup_-_Generic_v2.yml]: @erezh31 We always have doc review. @yaron-libman is going over it a bit later today. 

@mayagoldb I agree with Erez, please work together and update description before Yaron's review.

**Pull request closed without merging**